content/blog/2025-05-13-epel-10-kitten-v2.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "EPEL Packages Are Now Available for AlmaLinux Kitten 10 x86_64_v2!"
+type: blog
+author:
+ name: "Eduard Abdullin"
+ bio: "The Release and Automation Engineer."
+ image: /users/eduard-abdullin.jpg
+date: '2025-05-14'
+images:
+  - /blog-images/2025/2025-05-13-epel-10-kitten-v2.png
+post:
+    title: "EPEL Packages Are Now Available for AlmaLinux Kitten 10 x86_64_v2!"
+    image: /blog-images/2025/2025-05-13-epel-10-kitten-v2.png
+
+---
+
+AlmaLinux OS Foundation is happy to announce that EPEL support is now available on the **x86_64_v2** architecture for AlmaLinux Kitten 10!
+
+Many users rely on Fedora EPEL for extra tools and packages, and we know that AlmaLinux users want access to Fedora EPEL packages on every architecture that AlmaLinux supports. Fedora EPEL doesn't cover x86_64_v2,  since Fedora, CentOS, and RHEL don’t build for this architecture. [ALESCo decided](https://github.com/AlmaLinux/ALESCo/blob/master/rfcs/0001-build-fedora-epel-for-almalinux-and-almalinux-kitten-x86_64_v2.md) to build EPEL packages from Fedora’s source RPMs (SRPMs) to make x86_64_v2 just as functional and useful as any other AlmaLinux Kitten 10 architecture.
```

Comment:
```suggestion
Many users rely on Fedora EPEL for extra tools and packages, and we know that AlmaLinux users want access to Fedora EPEL packages on every architecture that AlmaLinux supports. Fedora EPEL doesn't cover x86_64_v2 on EL10  since CentOS Stream 10 and RHEL 10 aren't built for this architecture (they've moved up to a baseline of v3). [ALESCo decided](https://github.com/AlmaLinux/ALESCo/blob/master/rfcs/0001-build-fedora-epel-for-almalinux-and-almalinux-kitten-x86_64_v2.md) to build EPEL packages from Fedora’s source RPMs (SRPMs) to make x86_64_v2 just as functional and useful as any other AlmaLinux Kitten 10 architecture.
```